e2e/src/common.rs

```diff
@@ -57,7 +57,7 @@ pub fn create_accounts_with_funded_tokens(
     let accounts: Vec<H160> =
         web3.eth().accounts().wait().expect("get accounts failed")[..num_users].to_vec();
 
-    let tokens: Vec<IERC20> = (1..num_tokens)
+    let tokens: Vec<IERC20> = (0..num_tokens)
```

Comment:
Good catch! The 0th token used to be the OWL token. That's why I started at 1. But I think it got moved during refactorings. Your change looks right.